docs/_docs/02-ug-configuration.md

```diff
@@ -16,3 +16,6 @@ classes: wide
 | spark.hyperspace.explain.displayMode.highlight.endTag   | "" (An empty string)                                                                                    | Tag to mark ending of highlight portion in explain() output according to the display mode.            | 0.1.0         |
 | spark.hyperspace.index.lineage.enabled   | false                                                                                    | Add lineage column to index upon creation to track source data file for each index record. Lineage is required to handle deleted files in HybridScan, or to refresh an index in the incremental mode. Adding lineage will increase the size of the index, proportional to the number of distinct source data files the index is built on. | 0.3.0         |
 | spark.hyperspace.index.optimize.fileSizeThreshold   | 256MB                                                                                    | Threshold of size of index files in bytes to optimize. Files with size below this threshold are eligible for merge during index optimization.            | 0.3.0         |
+| spark.hyperspace.index.hybridscan.enabled   | false                                                                                  | Enable Hybrid Scan at query execution time. With this config, indexes with appended files and deleted files in their source dataset also can be candidate indexes with additional overhead. | 0.3.0         |
+| spark.hyperspace.index.hybridscan.delete.enabled   | false                                                                                    | (Temporal) Enable Hybrid Scan for deleted files. Lineage is required to handle deleted files in HybridScan.  | 0.3.0         |
```

Comment:
Lets be consistent in spelling HybridScan and pick either "Hybrid Scan" or "HybridScan". (i.e. with or wout a space in the middle).